sdk/storage/azure-storage-blobs/samples/blob_getting_started.cpp

```diff
@@ -40,4 +38,6 @@ void BlobsGettingStarted()
   blobClient.DownloadTo(buffer.data(), buffer.size());
 
   std::cout << std::string(buffer.begin(), buffer.end()) << std::endl;
+
+  return 0;
```

Comment:
nit. Instead of letting app go to unhandled exception, add try/catch to show catching Storage exception.

Reply:
I don't think it's a good habit to enclose many API calls with one try/catch block. It would be hard to tell which exactly API failed even if there was an exception, and the user wouldn't be able to figure out which APIs succeeded, and where to resume.

If we had one try/catch block for each API call, that would make the sample too lengthy, and our customers wouldn't be able to focus on the code we want to demonstrate.

Reply:
I'm thinking, maybe we should have an separate sample for how to catch storage exceptions.

Reply:
> It would be hard to tell which exactly API failed even if there was an exception

I would expect customers would turn logging to console on while trying to debug issues.  Then, the logging policy would tell what's the I/O request before throwing the exception.

But, yes,  this can be done later. Not a blocker for this PR